src/plugins/intel_gpu/tests/unit/test_cases/gemm_gpu_test.cpp

```diff
@@ -1313,16 +1313,22 @@ class gemm_gpu_tests: public ::testing::Test {
             output_shape_default = { M_SIZE, N_SIZE };
         } else if (num_dims == 3) {
             input0_shape_default = { BATCH_SIZE, M_SIZE, K_SIZE };
-            input1_shape_default = { BATCH_SIZE, K_SIZE, N_SIZE }; 
+            input1_shape_default = { BATCH_SIZE, K_SIZE, N_SIZE };
             output_shape_default = { BATCH_SIZE, M_SIZE, N_SIZE };
         } else if (num_dims == 4) {
             input0_shape_default = { BATCH_SIZE, 1, M_SIZE, K_SIZE};
-            input1_shape_default = { BATCH_SIZE, 1, K_SIZE, N_SIZE}; 
+            input1_shape_default = { BATCH_SIZE, 1, K_SIZE, N_SIZE};
             output_shape_default = { BATCH_SIZE, 1, M_SIZE, N_SIZE };
         }
     }
 
-    void test_transpose_matmul_f32(size_t num_dims, bool is_input_dynamic, bool is_caching_test, std::vector<size_t> BMKN, std::vector<int64_t> input0_order, std::vector<int64_t> input1_order) {
+    void test_transpose_matmul_f32(size_t num_dims,
+                                   bool is_input_dynamic,
+                                   bool is_caching_test,
+                                   std::vector<size_t> BMKN,
+                                   std::vector<int64_t> input0_order,
+                                   std::vector<int64_t> input1_order,
+                                   std::vector<int64_t> output_order = {}) {
```

Comment:
Note: There is test_transpose_matmul_transpose for output transpose test, I think we can remove it and merge to this. (can be done in separate work)